config/jobs/kubernetes/kops/templates/periodic.yaml.jinja

```diff
@@ -62,8 +62,7 @@
           {%- endif %}
           --test=kops \
           -- \
-          --ginkgo-args="--debug" \
-          --test-args="-test.timeout={{test_timeout}} -num-nodes=0" \
+          --test-args="-test.timeout={{test_timeout}} -num-nodes=0 --tooling=kops" \
```

Comment:
```suggestion
          --test-args="-test.timeout={{test_timeout}} -num-nodes=0" \
```

Reply:
If you look at the k/k PR closely, the flag already exists but it hasn't been used till my PR. Setting a value is fine and it will do something useful as soon as the change is merged.